src/proto/streams/recv.rs

```diff
@@ -586,6 +597,11 @@ impl Recv {
 
     pub fn consume_connection_window(&mut self, sz: WindowSize) -> Result<(), RecvError> {
         if self.flow.window_size() < sz {
+            trace!(
```

Comment:
Should these other error logs be emitted at debug!? it seems any time where the server fails a stream without the application taking action, we should at least log at debug...

Reply:
We can log these at debug as well --- I was scoping this change to protocol errors specifically, but I can change these to debug as well.